packages/os/metricdog.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Scheduled Metricdog Pings
+
+[Timer]
+# Don't run missed executions
+Persistent=false
+# Run 120 seconds after startup
+OnStartupSec=120
+# Run every 6 hours thereafter
+OnUnitActiveSec=21600
+# Don't fire at exactly the same second across machines started together.
+RandomizedDelaySec=10
```

Comment:
We probably want more jitter than this, maybe as much as an hour.

Reply:
Added jitter in this push [11b0927](https://github.com/bottlerocket-os/bottlerocket/compare/3db70e2f361b9cdd9306d75a1c2f6cf3d33eaadf..11b09274da45b6642d81f8164f2104b6157026e4)

Reply:
Oops set jitter to 3600 this time for one hour [92230f5](https://github.com/bottlerocket-os/bottlerocket/compare/560f33678840b377569ea0dbb2d01633767172ae..92230f53e1f7aefca7c67130b785565a6d172824)